### ISSUE TITLE
Feat: Migrate database to prisma for more deployment options

### ISSUE THREAD

**Issue by Cliftonz** [towfiqi/serpbear#95]
@towfiqi 

I would like to request that the database layer be rewritten in prisma this way could deploy the system not only with sqlite but with mongodb, planetscale, postgresql, mysql, and cockroachdb.

I would have no issue implementing this for the project however I do want to get the ok before I spend my time doing it. 

Thanks,
@cliftonz

**Comment by Cliftonz** [comment]
@towfiqi

**Comment by ryanovas** [comment]
Agreed. Docker volume mounts are fine when you're just putting stuff on your own servers but once you're using kube or production level stuff it's a hindrance
